ui/shared/components/buttons/SearchResultsLink.jsx

```diff
@@ -48,15 +48,14 @@ const ConnectedSearchResultsLink = connect(null, mapDispatchToProps)(SearchResul
 
 export default ConnectedSearchResultsLink
 
-const getGeneSearchProps = af => ({
-  inheritance: { mode: ANY_AFFECTED },
-  freqs: FREQUENCIES.filter(({ name }) => name !== THIS_CALLSET_FREQUENCY && name !== SV_CALLSET_FREQUENCY).reduce(
-    (acc, { name }) => ({ ...acc, [name]: { af } }), {},
-  ),
-})
-
-const INITIAL_GENE_SEARCH = getGeneSearchProps(0.1)
-const RARE_GENE_SEARCH = getGeneSearchProps(0.01)
+const INITIAL_GENE_SEARCH = {
+  inheritance: { mode: ANY_AFFECTED, filter: {} },
+  freqs: {
+    ...FREQUENCIES.filter(({ name }) => name !== 'topmed').reduce(
```

Comment:
all strings that are used more than once need to be constants. Similar to `THIS_CALLSET_FREQUENCY`, the constant needs to be defined in the constants file, used in the definition for the `FREQUENCIES`, and then imported here